src/plugins/data/common/search/aggs/buckets/range.ts

```diff
@@ -78,7 +78,9 @@ export const getRangeBucketAgg = ({ getFieldFormatsStart }: RangeBucketAggDepend
       return key;
     },
     getSerializedFormat(agg) {
-      const format = agg.params.field ? agg.params.field.format.toJSON() : {};
+      const format = agg.params.field
+        ? agg.aggConfigs.indexPattern.getFormatterForField(agg.params.field).toJSON()
+        : { id: undefined, params: undefined };
```

Comment:
why are `id` and `params` explicitly `undefined` here, as opposed to the empty object we had previously?

Reply:
typescript displays the following error - `Property 'id' does not exist on type '{}'`